charts/kubeclarity/values.yaml

```diff
@@ -203,12 +207,29 @@ kubeclarity-postgresql:
   ## Setup database name and password
   existingSecret: kubeclarity-postgresql-secret
   postgresqlDatabase: kubeclarity
+  # secretKey: postgresql-password
 
-  ## Enable security context
+  serviceAccount:
+    enabled: true
```

Comment:
default is `false` 
https://github.com/helm/charts/blob/master/stable/postgresql/values.yaml#L94

Reply:
It is good practice to specify a serviceAccount for a deployment, otherwise the `default` serviceAccount is used and this can then lead to someone unknowingly overassigning privilege at a later date. I'd suggest leave this in unless you really want me to remove it...?

Reply:
You are right, that is OK - we can keep it. Thanks!